GTG/plugins/urgency-color.gtg-plugin

```diff
@@ -2,7 +2,7 @@
 module=urgency_color
 name=Urgency Color
 short-description=Task urgency color-coding
-description=This plugin will calculate the urgency status of every of your currently active tasks and color-code it accordingly.\n\nColor code\n\nAssuming your settings are default:\nRed means you are within the last 30% of days between the start-date and due-date, or your task is overdue.\nYellow means your task's start-date has passed.\nGreen means your task's start-date is oncoming.
+description=This plugin will calculate the urgency status of every of your currently active tasks and color-code it accordingly.\n\nColor code\n\nAssuming your settings are default:\nGrey means your task is overdue.\nRed means you are within the last 30% of days between the start-date and due-date.\nYellow means your task's start-date has passed.\nGreen means your task's start-date is oncoming.
```

Comment:
`status of every [one] of your currently active tasks` - so I guess it's OK as it is.

Reply:
Ah, you're right

Reply:
Wait, we can't use "every of" it needs the "one" explicitly in there. That's what was tripping me, https://thegrammarexchange.infopop.cc/topic/every-of

Reply:
OK, so use the proper wording and fix that please. I hope the pot file gets updated soon, so I can finish and upload the translation.

Reply:
Why not change it in this PR since you are already changing that string? Once we merge this I'll push an update to the pot file

Reply:
I tried to do it as simple as possible, without touching git, to not get confused, since I don't work with it day to day and I have already too many things on my head to control console commands - making that patch was already a fork of my activities. So I would appreciate if someone else can do it.

Reply:
Sure, I'll get to it. BTW, do you have experience with translations in other projects? I'm thinking of using weblate.com to handle translations here

Reply:
Great, thanks! 

I made a few different translations over the years and never found a problem with tools - the most tedious thing is sending and branching. I have an account there, but don't remember if I used it already. Anyway - it's always easier to work in browser, so that could be nice.

Reply:
Hiya, @diegogangl if you are going to update the wording here — one other minor tweak: “…is oncoming.” at the end of the sentence doesn’t really make sense. It should be “upcoming.” Thanks :)